### PR TITLE
Add Python 3 and PyPy support.

### PR DESCRIPTION
**Pull request by jamadden**
Add tox.ini

Test configure.zcml

100% coverage.

### CODE REVIEW

File: src/zope/app/content/tests.py [hunk 1]
```diff
@@ -16,5 +16,21 @@
...
+        from zope.configuration import xmlconfig
+        xmlconfig.file('configure.zcml', package=zope.app.content)
+
+        component.getUtility(IVocabularyFactory, name="Content Types")
+
 def test_suite():
-    return doctest.DocTestSuite("zope.app.content")
+    return unittest.TestSuite((
+        doctest.DocTestSuite("zope.app.content"),
+        unittest.makeSuite(TestConfiguration),
```
Comment (mgedmin) [inline src/zope/app/content/tests.py]: `unittest.defaultTestLoader.loadTestsFromName(__name__)` in case somebody adds new test cases later?

**Review by mgedmin — APPROVED** [review]
LGTM

**Comment by jamadden** [comment]
Thanks. A PyPI release would be helpful, when convenient.

**Comment by mgedmin** [comment]
You now have PyPI rights for zope.app.content

**Comment by jamadden** [comment]
Thanks. Released as 4.0.

**Pull request merged**